jekyll/_cci2/faq.md

```diff
@@ -274,8 +274,13 @@ You can choose to pay for premium features per active user, compute, and optiona
 
 - Access to features, such as new machine sizes, are paid with a monthly fee of $15 per active user.
 - Compute is paid for monthly in credits for the machine size and duration you use.
-- Docker Layer Caching (DLC) is paid for with credits per usage, similar to
+- Docker Layer Caching (DLC) is paid for with credits per job, similar to
   compute credits.
+  
+#### Why does CircleCI have per-active-user pricing? 
+{:.no_toc}
+
+Credit usage covers access to compute. We prefer to keep usage costs as low as possible to encourage frequent job runs, which is the foundation of a good CI practice. Per-active-user fees cover access to platform features and job orchestration. This includes features like dependency caching, artifact caching, and workspaces, all of which speed up build times without incurring additional compute cost. Our per-active-user charge also allows us to provide support to all customers and deliver additional features like insights and orbs. 
```

Comment:
"...active-user charge also allows us to provide support to all customers and deliver additional features like insights and orbs." I'm not sure about this sentence since free plans still have these things and they don't pay for users. 

cc @rosaliebud